source-control/README.md

```diff
@@ -0,0 +1,100 @@
+Source code is stored on [Github](https://github.com/b2io/) using the [Git](http://git-scm.com/) source control system.
+
+## Git Configuration
+
+See the following **[`.gitconfig`](/source-control/.gitconfig)** as a starting point to setup the same configuration and aliases as the rest of the team. In Windows this file is located in the `C:\Users\<username>` directory.
+
+## Files to Ignore
+
+**[`.gitignore`](/source-control/.gitignore)** can be used to ignore common files such as IDE files. This global ignore file can be configured in the [`.gitconfig`](/source-control/.gitconfig) via
+
+```
+[core]
+excludesfile = ~/.gitignore
+```
+
+This is the preferred method of ignore non-project-specific files so that there is not a need to request a change to another repository's `.gitignore` file.
+
+The root directory of each repository should contain a `.gitignore` file that identifies the files that should be ignored for the project. These tend to vary based on project, but examples of common `.gitignore` files can be found: https://github.com/github/gitignore
+
+### Clients
+
+- Command line - The majority of the team uses command line exclusively. We tend to leverage the aliases in the **[`.gitconfig`](/source-control/.gitconfig)** fairly heavily..
+- [SourceTree](http://www.sourcetreeapp.com/)
+- [Visual Studio Code](https://code.visualstudio.com)
+
+### Mergetools
+
+- [KDiff3](http://kdiff3.sourceforge.net/)
+- [P4Merge](http://www.perforce.com/product/components/perforce-visual-merge-and-diff-tools)
+- [Visual Studio Code](https://code.visualstudio.com)
+
+## Workflow
+
+A new branch should be created for each issue/card. The feature branch should be pull requested back into the `master` branch, after peer approval the requester should rebase/squash the feature branch into `master`.
+
+1. Get latest from master: `$ git puff` (`$ git puff origin` if you happen to have multiple remotes)
```

Comment:
❓Let's add a `0.` step of "Ensure you're on the `master` branch: `$ git co master`?